Dockerfiles/dogstatsd/alpine/README.md

```diff
@@ -10,6 +10,7 @@ The following environment variables are supported:
   - `DD_HOSTNAME`: hostname to use for metrics
   - `DD_DOGSTATSD_SOCKET`: path to the unix socket to use instead of UDP. Must be in a `rw` mounted volume.
   - `DD_ENABLE_METADATA_COLLECTION`: whether to collect metadata (default is true, set to false only if running alonside an existing dd-agent)
+  - `DD_SITE`: Destination site for your metrics, can set to datadoghq.eu if required. Defaults to datadoghq.com.
```

Comment:
Are US3, US5 and gov also supported?

Reply:
I think so, there doesn't seem to be much documentation on the full list of available options. This was all I could find: https://docs.datadoghq.com/agent/troubleshooting/site/

Reply:
Ah just found this page https://docs.datadoghq.com/getting_started/site/

Reply:
@apigirl is there anything else I need to do to get this merged?